.cm/gitstream.cm

```diff
@@ -0,0 +1,64 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+# The `automations` section includes a list of automation that applies 
+# to the repository in which gitStream is installed. Each automation has an 
+# `if` key with a list of the necessary assertions, as well as a `run` key with a
+# list of all actions. All the listed assertions need to pass in order 
+# for the following actions to be executed (there is AND relation between conditions).
+
+# Each automation under the `automations` section is independent of the others. 
+# Every time a PR is opened or changed, the automation's conditions are evaluated (the `if`). 
+# The actions under `run` are executed one by one if all the conditions pass. 
+
+# Conditions consists of an expression, which are wrapped with double curly braces, and 
+# includes a context variable like `files` and filter functions like `length`. Filters 
+# functions are essentially functions that can be applied to context variables. They are 
+# called with a pipe operator (|) and can take arguments. Read more on https://docs.gitstream.cm
+
+config:
+    ignore_files:
+        - 'yarn.lock'
```

Comment:
Are there other ignore files that should be added here for android? Can probably get rid of yarn in there.

Reply:
done!